chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/DecertifiedDeveloperDTO.java

```diff
@@ -56,8 +40,24 @@ public void setNumMeaningfulUse(Long numMeaningfulUse) {
 		}
 	}
 	
-	public void addAcb(String acb){
-		this.oncacb.add(acb);
+	public void addAcb(Long acb){
+		this.acbList.add(acb);
```

Comment:
Would this work if someone got one of these objects by first calling the default constructor? Or if they passed in "null" as the list argument of the overloaded constructor?